docs/src/components/productMaterial/MaterialStyling.tsx

```diff
@@ -101,7 +101,10 @@ export const useResizeHandle = (
       if (target.current && dragging && clientX) {
         const objectRect = target.current.getBoundingClientRect();
         const newWidth = clientX - objectRect.left + dragOffset;
-        target.current.style.width = `clamp(${minWidth}, ${Math.floor(newWidth)}px, ${maxWidth})`;
+        target.current.style.setProperty(
```

Comment:
Using `style.setProperty` so there's no "mutation".

Reply:
This feels strange: https://github.com/facebook/react/issues/29832.